specification/maps/resource-manager/Microsoft.Maps/preview/2020-04-01-preview/maps-management.json

```diff
@@ -0,0 +1,1143 @@
+        "description": "Regenerate either the primary or secondary key for use with the Maps APIs. The old key will stop working immediately.",
+        "operationId": "Accounts_RegenerateKeys",
+        "x-ms-examples": {
+          "RegenerateKey": {
+            "$ref": "./examples/RegenerateKey.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "name": "keySpecification",
+            "in": "body",
+            "required": true,
+            "description": "Which key to regenerate:  primary or secondary.",
+            "schema": {
+              "$ref": "#/definitions/MapsKeySpecification"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful.",
+            "schema": {
+              "$ref": "#/definitions/MapsAccountKeys"
+            }
+          },
+          "404": {
+            "description": "The specified Account does not exist."
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Maps/operations": {
+      "get": {
+        "description": "List operations available for the Maps Resource Provider",
+        "operationId": "Accounts_ListOperations",
+        "x-ms-examples": {
+          "GetOperations": {
+            "$ref": "./examples/GetOperations.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null,
+          "itemName": "value"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful.",
+            "schema": {
+              "$ref": "#/definitions/MapsOperations"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Maps/accounts/{accountName}/privateAtlas/{privateAtlasName}": {
+      "put": {
+        "description": "Create or update a Private Atlas resource. Private Atlas resource will enable the usage of Azure resources to build a custom set of mapping data. It requires an account to exist before it can be created.",
+        "operationId": "PrivateAtlasCollection_CreateOrUpdate",
```

Comment:
>PrivateAtlasCollection [](start = 24, length = 22)

Is "PrivateAtlasCollection" the long form name of this concept?  Typically the prefix to the operationId is just the resource type

Reply:
Let me change it to PrivateAtlases, initially I had some reservations on the sound of "PrivateAtlases", but it's fine to keep it consistent